ai-service/src/main/java/org/libertybikes/ai/restclient/LibertyBikesClient.java

```diff
@@ -0,0 +1,40 @@
+/**
+ *
+ */
+package org.libertybikes.ai.restclient;
+
+import javax.enterprise.context.Dependent;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@Dependent
```

Comment:
the `@Dependent` anno can be removed -- it was made optional in MP Rest Client 1.1